src/wtc-modal-view.js

```diff
@@ -83,7 +83,14 @@ class Modal {
    */
   close() {
     if (this.state) {
-      this.modal.classList.remove(this.classNameOpen, this.optionalClass);
+      this.modal.classList.remove(this.classNameOpen);
+
+      if (this.optionalClass) {
+        if (typeof this.optionalClass === "string")
+          this.modal.classList.remove(this.optionalClass);
+        else if (typeof this.optionalClass === "object")
```

Comment:
I'd suggest changing this to 
`else if (this.optionalClass instanceof Array)`
Just in case someone passes some ridiculous value, 😄

Reply:
Done